workers/unity/Packages/io.improbable.gdk.deploymentlauncher/DeploymentConfigurations.cs

```diff
@@ -409,9 +409,9 @@ private bool ValidateSnapshotPath(out string message)
 
         private bool ValidateTag(string tag, out string message)
         {
-            if (!Regex.IsMatch(tag, "^[A-Za-z0-9][A-Za-z0-9_]{2,32}$"))
+            if (!Regex.IsMatch(tag, "^[A-Za-z0-9][A-Za-z0-9_]{1,31}$"))
```

Comment:
double checking to check what the actual validation is done for the regex

Reply:
This might be a good time to add some unit tests for these classes 😄

Reply:
it's on da list